lib/chainweb/request.ex

```diff
@@ -49,8 +51,9 @@ defmodule Kadena.Chainweb.Request do
     :segment_path,
     :query,
     :headers,
-    :body,
-    :encoded_query
+    :encoded_query,
+    :location,
+    body: ""
```

Comment:
@EdwinGuayacan why is this required?

Reply:
because in the `request/5` function of the `ClientDefault` it doesn't allow nil body:
![image](https://user-images.githubusercontent.com/80716239/211086746-2bfe650c-05ae-43d4-bad7-f86ac9ff700b.png)

Reply:
Got it, thanks!